conans/client/conf/config_installer.py

```diff
@@ -82,132 +102,140 @@ def _process_folder(folder, cache, output):
             elif f == "remotes.txt":
                 output.info("Defining remotes from remotes.txt")
                 _handle_remotes(cache, os.path.join(root, f))
-            elif f == "README.md":
-                output.info("Skip README.md")
-            elif f == "LICENSE.txt":
-                output.info("Skip LICENSE.txt")
             else:
+                # This is ugly, should be removed in Conan 2.0
+                if root == folder and f in ("README.md", "LICENSE.txt"):
```

Comment:
Not sure about this, Bincrafters uses LICENSE.md. Also, some projects use COPYING

Reply:
This wasn't added in this PR, I just kept the existing functionality. In fact, they will be only skipped now if they are in the root folder, but not for subfolders. So all files will be copied from subfolders.